unison-src/tests/soe.u

```diff
@@ -61,7 +61,8 @@ Monoid.orElse m = cases
 
 merge : (a -> a -> Boolean) -> [a] -> [a] -> [a]
 merge lte a b =
-  go acc a b = match List.at 0 a with
+  use List at
+  go acc a b = match at 0 a with
```

Comment:
Same here; why change the test at all?

Reply:
In both these files, there was an unqualified `at` or two in the function, and there wasn't a `use` to pin down what `at` it was referring to (I think you need to view more than just the immediately surrounding lines since the unqualified usage might be later in the function). I could have either changed these all to `List.at` or what I instead did was added an explicit `use List at` and then changed all the use sites to in the block to use that unqualified `at`.